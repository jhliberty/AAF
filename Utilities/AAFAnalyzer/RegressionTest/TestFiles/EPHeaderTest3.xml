<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE aaf-file SYSTEM "EPDerivationChain.dtd">
<aaf-file operational-pattern="invalid">
<!-- Make sure the file is not empty -->
 <top-level name="Invalid Header">
  <timeline-mob-slot edit-rate-denominator="1" edit-rate-numerator="1">
   <eoc length="0" track-type="none"/>
  </timeline-mob-slot>
 </top-level>
</aaf-file>
