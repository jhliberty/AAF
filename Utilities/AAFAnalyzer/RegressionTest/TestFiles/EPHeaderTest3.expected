        Name:   Edit Protocol Header Test
        Desc:   Verify the AAF file header is valid.
        Result: Failed
        Reason: 
        Covered Requirements: None
        Noted Requirements: None
        Passing Requirements: None
        Info Requirements: None
        Warning Requirements: None
        Failing Requirements: REQ_EP_255

            Name:   Protocol Label
            Desc:   Header::OperationalPattern should be equal to OpEditProtocol.
            Node:   1 (Header)
            Result: Failed
            Reason: Header::OperationalPattern property is not equal to OpEditProtocol.
            Covered Requirements: None
            Noted Requirements: None
            Passing Requirements: None
            Info Requirements: None
            Warning Requirements: None
            Failing Requirements: REQ_EP_255

